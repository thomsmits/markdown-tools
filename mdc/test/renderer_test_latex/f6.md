# Chapter
## Slide

Ersetzung von ... durch entsprechende (...).
  * Ersetzung von ... durch entsprechende (...).
    - und hier ... durch (...), ...
    - Neugierde, Faulheit, ...
