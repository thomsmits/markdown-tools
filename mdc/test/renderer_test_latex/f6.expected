# Chapter
## Slide

Ersetzung von \dots{} durch entsprechende (\dots{}).\vspace{0.1mm}

\begin{ul1}
  \item Ersetzung von \dots{} durch entsprechende (\dots{}).
  \begin{ul2}
    \item und hier \dots{} durch(\dots{}), \dots{}
    \item Neugierde, Faulheit, \dots{}
  \end{ul2}
\end{ul1}
